src/main/java/net/neoforged/neoforge/common/world/BiomeModifiers.java

```diff
@@ -180,18 +186,19 @@ public Codec<? extends BiomeModifier> codec() {
      * }
      * </pre>
      *
-     * @param biomes      Biomes to add mob spawns to.
-     * @param entityTypes EntityTypes to remove from spawn lists.
+     * @param biomes      Biomes to remove mob spawns from.
+     * @param entityTypes {@linkplain EntityType}s to remove from spawn lists.
      */
-    public record RemoveSpawnsBiomeModifier(HolderSet<Biome> biomes, HolderSet<EntityType<?>> entityTypes) implements BiomeModifier {
+    public record RemoveSpawnsBiomeModifier(HolderSet<Biome> biomes,
+            HolderSet<EntityType<?>> entityTypes) implements BiomeModifier {
         @Override
         public void modify(Holder<Biome> biome, Phase phase, ModifiableBiomeInfo.BiomeInfo.Builder builder) {
             if (phase == Phase.REMOVE && this.biomes.contains(biome)) {
                 MobSpawnSettingsBuilder spawnBuilder = builder.getMobSpawnSettings();
-                for (MobCategory category : MobCategory.values()) {
+                Arrays.stream(MobCategory.values()).forEach(category -> {
                     List<SpawnerData> spawns = spawnBuilder.getSpawner(category);
                     spawns.removeIf(spawnerData -> this.entityTypes.contains(BuiltInRegistries.ENTITY_TYPE.wrapAsHolder(spawnerData.type)));
-                }
+                });
```

Comment:
please don't change loops to streams (especially loops over arrays)

Reply:
Will do